### ISSUE TITLE
[Staging and Production] Too many error message with similar content

### ISSUE THREAD

**Issue by ginavuu** [CenterForOpenScience/osf.io#2616]
Steps
1. In staging, go to https://staging.osf.io/usqw2/
2. Fork the project

Expected 
The add-ons not to transfer and receive one compiled error message.

Actual
Receive several errors message, with some repeating messages, that takes up 1/3 of the page.
![part 1](https://cloud.githubusercontent.com/assets/11032617/7435688/d7555e30-f013-11e4-815e-c29dcb3acae9.PNG)
![part 2](https://cloud.githubusercontent.com/assets/11032617/7435725/376d3856-f014-11e4-9fbe-937e014814e0.PNG)

**Comment by chennan47** [comment]
Agreed. Perhaps just shorten the message to be All addons authorization not copied to forked project. You may authorize this fork on the Settings page. Also in the google drive warning, there is no warning between "Settings" and "page"

**Comment by ginavuu** [comment]
@chennan47 Good catch on the google drive, I was too overwhelm to catch that.

**Comment by lbanner** [comment]
Yeah -- good catch both of you -- I definitely think we can improve, and work on consistency here! Thanks.

**Comment by brianjgeiger** [comment]
When the growlbox was added, it was a quick drop-in replacement for a couple of other alert types. The most common request, which was out of scope at the time, was to be able to buffer multiple alerts that would happen at once. It might be good to have a general solution for this, and not just to fix it in this one instance.

**Comment by caileyfitz** [comment]
@nanchen47 Google Drive also has another issue in that it only refers to a "fork" instead of a "forked project" or "forked component" when authorization is transferred. See the fifth alert:
![screen shot 2015-05-06 at 10 25 35 am](https://cloud.githubusercontent.com/assets/5975444/7495654/4402ea30-f3dd-11e4-8286-7bc1f7f8f3d2.png)
